src/main/java/com/example/weatherwaveapi/service/OpenWeatherService.java

```diff
@@ -116,7 +119,6 @@ private WeatherOpenApiContainer getForecastContainerByCity(String city) {
     private WeatherApiResponse convertContainer(WeatherOpenApiContainer container) {
         return Optional.ofNullable(container.errorMessage())
                 .map(error -> WeatherApiResponse.builder()
-                        .success(false)
```

Comment:
why was this line removed?

Reply:
let's leave in all replies a field with a flag indicating whether everything went well or not

Reply:
Last time you said me to remove it because in json we don't have this field

Reply:
let's leave it anyway.